src/huggingface_hub/hf_api.py

```diff
@@ -264,6 +261,41 @@ def __str__(self):
         return r
 
 
+class SpaceInfo:
+    """
+    Info about a Space accessible from huggingface.co
+    """
```

Comment:
this docstring needs to be expanded to document the `__init__` args.

Reply:
This one is really an internal "dataclass" like feature, almost like a dict. Do you think it's still useful to document the attributes in the docstring for this?

Reply:
It's not just for our end users. Things become more maintainable for other contributors if they're better documented. That's why I ended up writing documentation for `_request_with_retry` [here](https://github.com/huggingface/huggingface_hub/pull/731/files#diff-765507f4602931b08aa20d0177611d5da7d5c7805882b0a79c81b5ed72c9472aR301) for example.